modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusRoutes.scala

```diff
@@ -1,24 +1,61 @@
 package org.tessellation.sdk.infrastructure.consensus
 
 import cats.effect.Async
-import cats.syntax.all._
-import cats.{Order, Show}
+import cats.syntax.contravariantSemigroupal._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import cats.syntax.order._
+import cats.{Applicative, Order}
 
-import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.domain.cluster.services.Cluster
 import org.tessellation.sdk.infrastructure.consensus.registration.RegistrationResponse
 
-import io.circe.{Decoder, Encoder}
+import eu.timepit.refined.auto._
+import io.circe.Encoder
 import org.http4s.HttpRoutes
 import org.http4s.circe.CirceEntityCodec._
 import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
 
-class ConsensusRoutes[F[_]: Async: KryoSerializer, Key: Order: Show: Encoder: Decoder](
-  storage: ConsensusStorage[F, _, Key, _]
+class ConsensusRoutes[F[_]: Async, Key: Encoder: Order](
+  cluster: Cluster[F],
+  storage: ConsensusStorage[F, _, Key, _],
+  selfId: PeerId
 ) extends Http4sDsl[F] {
 
-  val routes: HttpRoutes[F] = HttpRoutes.of[F] {
-    case GET -> Root / "consensus" / "registration" =>
+  private val prefixPath = "/consensus"
+
+  private val p2p: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "registration" =>
       storage.getOwnRegistration.map(RegistrationResponse(_)).flatMap(Ok(_))
   }
 
+  private val public: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest/peers" =>
+      for {
+        clusterPeers <- cluster.info
+        registeredPeers <- allRegisteredPeers
+        peers = clusterPeers.filter(p => registeredPeers.contains(p.id))
+        result <- Ok(peers)
+      } yield result
```

Comment:
I see there is no information about consensus ordinal, but it should be
![image](https://user-images.githubusercontent.com/4959931/198548593-162905c0-aeb2-4503-a2b4-fb838d7d8a13.png)